common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -161,6 +161,45 @@ object CometConf extends ShimCometConf {
       defaultValue = false,
       notes = Some("stddev is slower than Spark's implementation"))
 
+  val COMET_EXEC_PROJECT_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_PROJECT, defaultValue = false)
+  val COMET_EXEC_FILTER_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_FILTER, defaultValue = false)
+  val COMET_EXEC_SORT_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_SORT, defaultValue = false)
+  val COMET_EXEC_LOCAL_LIMIT_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_LOCAL_LIMIT, defaultValue = false)
+  val COMET_EXEC_GLOBAL_LIMIT_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_GLOBAL_LIMIT, defaultValue = false)
+  val COMET_EXEC_BROADCAST_HASH_JOIN_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_BROADCAST_HASH_JOIN, defaultValue = false)
+  val COMET_EXEC_BROADCAST_EXCHANGE_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_BROADCAST_EXCHANGE, defaultValue = false)
+  val COMET_EXEC_HASH_JOIN_DISABLED: ConfigEntry[Boolean] =
+    createExecDisabledConfig(OPERATOR_HASH_JOIN, defaultValue = false)
+  val COMET_EXEC_SORT_MERGE_JOIN_DISABLED: ConfigEntry[Boolean] =
```

Comment:
For example, I use this to test SortMergeJoin in #553.

Reply:
It seems strange to have both an `enabled` and `disabled` for each operator. What do we do if both are true or both are false?

In https://github.com/apache/datafusion-comet/pull/840 I had suggested another approach, which I wasn't too happy with, and @parthchandra had added another suggestion in there

Reply:
In `isCometOperatorEnabled`, currently `disabled` configs have higher priority.

But I agree that it would make confusion.

Another proposal would be to disable the operator if `enabled` config is false even `COMET_EXEC_ALL_OPERATOR_ENABLED` is true.

Reply:
I.e., the `enabled` can be used to enable/disable specified operator.

Reply:
Hmm, it doesn't make sense too. These `enabled` configs are false by default. Which will disable these configs.

Reply:
> Hmm, it doesn't make sense too. These `enabled` configs are false by default. Which will disable these configs.

In PR I had changed them all to default to true, so we can disable by setting to false

Reply:
I had also removed the `ALL` option

Reply:
> I had also removed the `ALL` option

You mean `COMET_EXEC_ALL_OPERATOR_ENABLED`?

Reply:
I created a new PR with just the config change
https://github.com/apache/datafusion-comet/pull/855